src/components/Button/Button.jsx

```diff
@@ -42,7 +42,7 @@ Button.propTypes = {
   /** Any additional classes to be applied to the button */
   className: propTypes.string,
   /** Children elements. */
-  children: propTypes.string,
+  children: propTypes.node,
 };
```

Comment:
can we just add an `icon="<icon_kind>"` prop to the Button component that when present add the icon in the button? This will help keep icon usage within Buttons consistent. Without it you could just add an icon anywhere in the button (before or after the button text)